signals/content/impersonation_msft.yml

```diff
@@ -0,0 +1,36 @@
+name: "Impersonation: Microsfot"
```

Comment:
```suggestion
name: "Impersonation: Microsoft"
```

Reply:
I was showing the power of levenshtein distance :trollface: